pkg/cmd/cli/cmd/env.go

```diff
@@ -260,7 +265,19 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 				return nil
 			}
 			for _, c := range containers {
+				oldEnv := c.Env
+				for _, env := range remove {
+					_, ok := findEnv(c.Env, env)
+					if !ok {
+						infosMsgs[i] = append(infosMsgs[i], fmt.Sprintf("environment variable %q not found in %s container\n", env, c.Name))
```

Comment:
So if something has multiple containers, one of which has an envvar, and you remove the envvar, you'll always get a message saying the other container didn't have that var? That doesn't seem necessary.